src/Resources/Locales/de_DE.axaml

```diff
@@ -402,8 +422,12 @@
   <x:String x:Key="Text.Period.YearsAgo" xml:space="preserve">Vor {0} Jahren</x:String>
   <x:String x:Key="Text.Preference" xml:space="preserve">Einstellungen</x:String>
   <x:String x:Key="Text.Preference.AI" xml:space="preserve">OPEN AI</x:String>
-  <x:String x:Key="Text.Preference.AI.Server" xml:space="preserve">Server</x:String>
+  <x:String x:Key="Text.Preference.AI.AnalyzeDiffPrompt" xml:space="preserve">Analysierung des Diff Befehl</x:String>
   <x:String x:Key="Text.Preference.AI.ApiKey" xml:space="preserve">API Schlüssel</x:String>
+  <x:String x:Key="Text.Preference.AI.GenerateSubjectPrompt" xml:space="preserve">Generiere Nachricht Befehl</x:String>
+  <x:String x:Key="Text.Preference.AI.Name" xml:space="preserve">Name</x:String>
+
+  <x:String x:Key="Text.Preference.AI.Server" xml:space="preserve">Server</x:String>
   <x:String x:Key="Text.Preference.AI.Model" xml:space="preserve">Modell</x:String>
```

Comment:
The order should be `Model`, `Name`, `Server` as it is in the English translation.
```suggestion
  <x:String x:Key="Text.Preference.AI.Model" xml:space="preserve">Modell</x:String>
  <x:String x:Key="Text.Preference.AI.Name" xml:space="preserve">Name</x:String>
  <x:String x:Key="Text.Preference.AI.Server" xml:space="preserve">Server</x:String>

```